networkdb/cluster.go

```diff
@@ -306,6 +311,7 @@ func (nDB *NetworkDB) reapState() {
 
 func (nDB *NetworkDB) reapNetworks() {
 	nDB.Lock()
+	defer nDB.Unlock()
```

Comment:
it's technically faster to not use `defer`, I'd suggest keeping it at the bottom since the function only has 1 exit path

Reply:
k will change